data/en/cfinsert.json

```diff
@@ -12,17 +12,17 @@
 		{"name":"username","description":"Overrides username specified in ODBC setup.","required":false,"default":"","type":"string","values":[]},
 		{"name":"password","description":"Overrides password specified in ODBC setup.","required":false,"default":"","type":"string","values":[]},
 		{"name":"formfields","description":"Comma-delimited list of form fields to insert. If not\n specified, all fields in the form are included.\n\n If a form field is not matched by a column name in the\n database, CFML throws an error.\n\n The database table key field must be present in the form.\n It may be hidden.","required":false,"default":"","type":"string","values":[]},
-		{"name":"providerdsn","description":"Deprecated","required":false,"default":"","type":"string","values":[]},
-		{"name":"dbtype","description":"Deprecated","required":false,"default":"","type":"string","values":[]},
-		{"name":"dbname","description":"Deprecated","required":false,"default":"","type":"string","values":[]},
-		{"name":"dbserver","description":"Deprecated","required":false,"default":"","type":"string","values":[]},
-		{"name":"provider","description":"Deprecated","required":false,"default":"","type":"string","values":[]}
+		{"name":"providerdsn","description":"","required":false,"default":"","type":"string","values":[]},
```

Comment:
It's probably ok and useful to keep `Deprecated` in the description even though it is added to the engine notes.

Reply:
Shouldn't we add a real description?

From the moment we start adding `engine` structure to `param` we could add a `deprecated` key there.